### PR TITLE
Add listLoanSchemes rpc

### PR DESCRIPTION
**Pull request by jingyi2811**
#### What kind of PR is this?:

/kind feature

#### What this PR does / why we need it:

Add listLoanSchemes rpc

#### Which issue(s) does this PR fixes?:

Fixes part of #488

### CODE REVIEW

**Review by surangap — APPROVED** [review]

**Review by Jouzo — DISMISSED** [review]

**Review by siradji — APPROVED** [review]

**Review by canonbrother — APPROVED** [review]

File: website/docs/jellyfish/api/loan.md [hunk 1]
```diff
@@ -33,3 +33,20 @@ interface UTXO {
...
+## listLoanSchemes
+
+List all available loan schemes.
+
+```ts title="client.loan.listLoanSchemes()"
+interface loan {
+  listLoanSchemes (): Promise<LoanSchemeResult[]>
+}
+
+interface LoanSchemeResult{
```
Comment (fuxingloh) [inline website/docs/jellyfish/api/loan.md:46]:
```suggestion
interface LoanSchemeResult {
```

File: packages/jellyfish-api-core/src/category/loan.ts [hunk 2]
```diff
@@ -26,6 +26,15 @@ export class Loan {
   async createLoanScheme (scheme: CreateLoanScheme, utxos: UTXO[] = []): Promise<string> {
     return await this.client.call('createloanscheme', [scheme.minColRatio, scheme.interestRate, scheme.id, utxos], 'number')
   }
+
+  /**
+   * List all available loan schemes.
+   *
+   * @return {Promise<LoanSchemeResult[]>}
+   */
+  async listLoanSchemes (): Promise<LoanSchemeResult[]> {
```
Comment (fuxingloh) [inline packages/jellyfish-api-core/src/category/loan.ts:35]: No pagination?

I guess we might not ever need them.

File: packages/jellyfish-api-core/src/category/loan.ts [hunk 3]
```diff
@@ -38,3 +47,10 @@ export interface UTXO {
   txid: string
   vout: number
 }
+
+export interface LoanSchemeResult{
```
Comment (fuxingloh) [inline packages/jellyfish-api-core/src/category/loan.ts:51]:
```suggestion
export interface LoanSchemeResult {
```

**Review by fuxingloh — APPROVED** [review]

File: packages/jellyfish-api-core/src/category/loan.ts (hunk 2 above)
Comment (jingyi2811) [reply packages/jellyfish-api-core/src/category/loan.ts:35]: Ya, no pagination

**Pull request merged**